dashboard/src/components/IssueDetails/IssueDetails.tsx

```diff
@@ -168,10 +176,13 @@ export const IssueDetails = ({
   }, [data, formatMessage, getCulpritValue]);
 
   const sectionsData: ISection[] = useMemo(() => {
-    return [...generalSections, logspecSection, miscSection].filter(
-      section => section !== undefined,
-    );
-  }, [generalSections, logspecSection, miscSection]);
+    return [
+      ...generalSections,
+      firstIncidentSection,
+      logspecSection,
+      miscSection,
+    ].filter(section => section !== undefined);
```

Comment:
```suggestion
    ].filter(section => !!section);
```

tha way you acknowledge for `null`s also